### PR TITLE
Fix exporting to Scrivener when we need to delete an existing file

### PR DESCRIPTION
**Pull request by Quiescent**

### CODE REVIEW

File: src/app/_index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import path from 'path'
 import { store } from 'store'
 import { ipcRenderer } from 'electron'
 import { dialog, getCurrentWindow } from '@electron/remote'
+import { rm } from 'fs'
```
Comment (cameronsutter) [inline src/app/_index.js]: do we need this anymore?

File: src/app/_index.js [hunk 2]
```diff
@@ -101,6 +102,7 @@ ipcRenderer.on('export-file-from-menu', (event, { type }) => {
     logger,
     exportSaveDialog,
     MPQ,
+    rm,
```
Comment (cameronsutter) [inline src/app/_index.js]: wouldn't we want to use the rmRF here instead?

**Review by cameronsutter — COMMENTED** [review]
Just a couple questions. I'm probably just not following the code well enough

File: src/app/_index.js (hunk 1 above)
Comment (Quiescent) [reply src/app/_index.js]: You're right.  Good catch.  It should fail when we export via the file menu at the moment.
I'll double-check that that's the case for sanity's sake.

File: src/app/_index.js (hunk 2 above)
Comment (Quiescent) [reply src/app/_index.js]: Yup

**Pull request merged**